man/ostree.repo-config.xml

```diff
@@ -416,6 +416,14 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>boot_counting</varname></term>
+        <listitem><para>Integer value controlling the number of maximum boot attempts. The boot counting data is stored in the name of the
+        boot loader entry. A boot loader entry file name may contain a plus (+) followed by a number. This may optionally be followed by
+        a minus (-) followed by a second number. The dot (.) and file name suffix (conf or efi) must immediately follow.
```

Comment:
Feel like a link to the spec here would be helpful.

Reply:
Added a link to the sepc